container-host-files/opt/scf/bin/klog.sh

```diff
@@ -24,56 +24,107 @@ fi
 NS=${1-scf}
 DONE="${KLOG}/${NS}/done"
 
+# Prevent bail out
```

Comment:
Missing punctuation here and below. There are also a couple comments inside the function. For consistency, how about putting them out of the function?

Reply:
Done with commit 51b44ecf987cdde4f6676f0e17e92bc2dda5d189